cli/popper/scm.py

```diff
@@ -183,6 +183,12 @@ def get_default_branch(r):
         else:
             url += '/'
 
+        auth_token = os.getenv('GITHUB_API_TOKEN')
+
+        if(auth_token is not None):
+            if('@' not in url):
```

Comment:
this conditional will result in not being able to work with URLs of the form `git@github.com`, which is OK but we need to add this to the documentation.

Reply:
Yes, I can add this to documentation to make it clear to the user.